tools/StaticAnalysis/Exceptions/Az.FrontDoor/SignatureIssues.csv

```diff
@@ -18,3 +18,4 @@
 "Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewFrontDoorRulesEngineActionObject","New-AzFrontDoorRulesEngineActionObject","1","8510","Cmdlet 'New-AzFrontDoorRulesEngineActionObject' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewFrontDoorRulesEngineMatchConditionObject","New-AzFrontDoorRulesEngineMatchConditionObject","1","8100","New-AzFrontDoorRulesEngineMatchConditionObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewFrontDoorRulesEngineRuleObject","New-AzFrontDoorRulesEngineRuleObject","1","8100","New-AzFrontDoorRulesEngineRuleObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Az.FrontDoor","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewFrontDoorRulesEngineActionObject","New-AzFrontDoorRulesEngineActionObject","1","8700","Parameter set '__AllParameterSets', 'ByFieldsWithRedirectParameterSet' of cmdlet 'New-AzFrontDoorRulesEngineActionObject' have the same mandatory parameters, and both of them are not default parameter set which may cause confusion.","Merge these parameter sets into one parameter set."
```

Comment:
That sounds like a StaticAnalysis issue, as I've already mark ByFieldsWithRegularActionParameterSet as the default parameter set.